UI/src/app/components/pages/mapping/mapping.component.html

```diff
@@ -1,56 +1,24 @@
 <div class="app-mapping">
-  <div class="export-actions">
-    <button
-      mat-icon-button
-      matTooltip="Download ziped mappings"
-      aria-label="generate mapping xmls"
-      (click)="generateMappingJson()"
-    >
-      <mat-icon>file_download</mat-icon>
-    </button>
-    <button
-      mat-icon-button
-      matTooltip="Preview mappings"
-      aria-label="preview mapping xmls"
-      (click)="previewMapping()"
-    >
-      <mat-icon>play_circle_filled_white</mat-icon>
-    </button>
-    <button
-      mat-icon-button
-      matTooltip="Delete all mappings"
-      aria-label="delete all mappings"
-      (click)="wipeAllMappings()"
-    >
-      <mat-icon>delete</mat-icon>
-    </button>
-  </div>
   <section>
     <main #maincanvas class="main">
       <app-area area="source" class="area-source">
-        <ng-container *ngFor="let table of source; trackBy: trackByFn">
-          <ng-container *ngIf="table.visible">
-            <app-panel
+        <mat-tab-group (selectedIndexChange)="onTabIndexChanged($event, source, 'source')">
+          <mat-tab
+            label="{{ table.name | prettyName }}"
```

Comment:
let's use unified attribute one-way bindings style if it's possible:
```ts
[label]="table.name | prettyName"
```

Reply:
done